ignite/pkg/xast/function.go

```diff
@@ -483,3 +484,125 @@ func structToBlockStmt(code string) (ast.Expr, error) {
 func toStruct(code string) string {
 	return fmt.Sprintf(`struct {}{ %s }`, strings.TrimSpace(code))
 }
+
+// ModifyCaller replaces all arguments of a specific function call in the given content.
+// The callerExpr should be in the format "pkgname.FuncName" or just "FuncName".
+// The modifiers function is called with the existing arguments and should return the new arguments.
+func ModifyCaller(content, callerExpr string, modifiers func([]string) ([]string, error)) (string, error) {
```

Comment:
This feature can be part of the `ModifyFunction` function, and we can use it like this:

```
content, err = xast.ModifyFunction(
	content,
	"main",
	xast.ModifyCaller(content, callerExpr, modifier),
)
```

wdyt?

Reply:
If we keep it as it is, maybe we can add some options like modify only the first or the last one from the function

Reply:
I prefer another function to make it more clear tbh. For more modifiers this can indeed be done eventually. 
Happy to do a follow up when we need those.